build.gradle

```diff
@@ -148,39 +148,45 @@ def assemblyDeps = [downloadAndInstallJRuby, assemble] + subprojects.collect {
   it.tasks.findByName("assemble")
 }
 
-task installBundler(dependsOn: assemblyDeps) {
-  outputs.files file("${projectDir}/vendor/bundle/jruby/2.5.0/bin/bundle")
+tasks.register("installBundler"){
```

Comment:
Layout notes, 
- alignment of the `dependsOn`
- space between `)` and `{`; there some other below of the same type